modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -47,34 +43,38 @@ anypoint.platform.gatekeeper=flexible
 
 ==== Disabled
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process.
+When you set this value, Gatekeeper does not apply any security. For that reason, the API might receive unwanted traffic during the period when the policies are applied.
 
 ==== True
 
-When this value is set, the default security level is Strict. This value is available only for backward compatibility in Mule 3.x. Use ‘strict’ or ‘flexible’ instead. `true` is unavailable in Mule 4.x
+When you set this value, the default security level is  set to `Strict`. In Mule 3.x, this value is available only for backward compatibility. Therefore, use ‘strict’ or ‘flexible’ instead for all other scenarios. The value `true` is unavailable in Mule 4.x releases.
 
 ==== Strict
 
-When this value is established, Gatekeeper security is set on its maximum security level.
+When you set this value, Gatekeeper security is set on its maximum security level.
 
-When the Mule Runtime is (re)started, the API is blocked and returns a 503 HTTP status code, until policies for each tracked resource are correctly downloaded and applied. If connectivity with API Platform is lost when Gatekeeper is engaged, the tracked resource will be blocked at least, until connection is re-established.
+When the Mule instance is (re)started, the API is blocked and a "503 - Service Unavailable" HTTP status code is returned until policies for each tracked resource are correctly applied. If the connectivity to Anypoint Platform is lost when Gatekeeper is engaged, the tracked resource is blocked until the connection is reestablished.
 
 ==== Flexible
 
 When this value is established, Gatekeeper security is set on its intermediate security level.
 
-When the Mule Runtime is (re)started, the API is blocked and returns a 503 HTTP status code, until policies for each tracked resource are correctly downloaded and applied. If connectivity with API Platform is lost when Gatekeeper is engaged, a local cache is used to get policies while connectivity is lost. The tracked resource will be blocked at least, until connection is re-established, if no cached policy is stored locally.
+When the Mule instance is (re)started, the API is blocked and a "503 - Service Unavailable" HTTP status code is returned until policies for each tracked resource are correctly applied. If the connectivity to Anypoint Platform is lost when Gatekeeper is engaged, a local cache is used to download and apply the policies during the outage. The tracked resource is blocked at least until the connection is reestablished.
```

Comment:
Last sentence should clarify that is blocked until connection is reestablished only if cache is not present.
Also, there is no download when cache is used, so I would just say local cache is used to apply the policies during the outage

Reply:
Done! Thanks!